SConstruct

```diff
@@ -78,6 +78,8 @@ opts.Add(
     )
 )
 
+opts.Add(BoolVariable("use_asan", "Use address sanitizer (ASAN)", False))
```

Comment:
This should be marked as Windows and MSVC only (or support other platforms as well).

Reply:
I thought these changes already support GCC/Clang. The main build system in Godot does the same: add the `/INFERSANLIBS` that non-MSVC will ignore and also `/fsanitize=address` that both MSVC and others will honor.

I can be deeply wrong, though.

Reply:
Not, both of these are MSVC only, and only defined in the MSVC code path of the main repo. For GCC/Clang it's:

```
env.Append(CCFLAGS=["-fsanitize=address,pointer-subtract,pointer-compare"])
env.Append(LINKFLAGS=["-fsanitize=address"])
```

Reply:
For MinGW it seems like it's not used in the main repo at all (only for Linux, macOS, and Web), not sure if it's not supported or just missing.

Reply:
I see. I will then just keep all this for MSVC for now.